great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -722,11 +722,6 @@ def _batch_id_to_batch_identifier_display_name_map(
         }
 
 
-@measure_execution_time(
-    execution_time_holder_object_reference_name="data_assistant_result",
-    execution_time_property_name="profiler_execution_time",
-    pretty_print=False,
-)
```

Comment:
If we need to profile performance, we can add temporary code to do so - I don't think we need to decorate each time.

If you disagree, lmk and I can bring this back